### PR TITLE
added folder with types

### PR DESCRIPTION
**Pull request by Afanasi0101**

### CODE REVIEW

File: src/components/contentWrapper.js [hunk 1]
```diff
@@ -7,9 +7,9 @@ function primaryAction() {
 
 function createContentWrapper() {
   const contentWrapper = document.createElement('div');
-  const primaryBtn = createBtn('play the game', 'primary', primaryAction);
+  const primaryBtn = createBtn('play the game', 'baseLargeBtn', primaryAction);
```
Comment (aleksanderwalczuk) [inline src/components/contentWrapper.js:10]: Then use the variable referring to that key in this function call.

File: src/components/icon.js [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
-  <path d="M4.462 19.462c.42-.419.753-.89 1-1.394.453.213.902.434 1.347.661a6.743 6.743 0 01-1.286 1.794.75.75 0 11-1.06-1.06z" />
-</svg>
-`,
-  user: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6 user_icon">
-  <path fill-rule="evenodd" d="M7.5 6a4.5 4.5 0 119 0 4.5 4.5 0 01-9 0zM3.751 20.105a8.25 8.25 0 0116.498 0 .75.75 0 01-.437.695A18.683 18.683 0 0112 22.5c-2.786 0-5.433-.608-7.812-1.7a.75.75 0 01-.437-.695z" clip-rule="evenodd" />
-</svg>
-`,
+  hatIcon,
+  userIcon,
 };
```
Comment (aleksanderwalczuk) [inline src/components/icon.js:6]: Same here.

File: src/types.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<path d="M13.06 15.473a48.45 48.45 0 017.666-3.282c.134 1.414.22 2.843.255 4.285a.75.75 0 01-.46.71 47.878 47.878 0 00-8.105 4.342.75.75 0 01-.832 0 47.877 47.877 0 00-8.104-4.342.75.75 0 01-.461-.71c.035-1.442.121-2.87.255-4.286A48.4 48.4 0 016 13.18v1.27a1.5 1.5 0 00-.14 2.508c-.09.38-.222.753-.397 1.11.452.213.901.434 1.346.661a6.729 6.729 0 00.551-1.608 1.5 1.5 0 00.14-2.67v-.645a48.549 48.549 0 013.44 1.668 2.25 2.25 0 002.12 0z" />
+<path d="M4.462 19.462c.42-.419.753-.89 1-1.394.453.213.902.434 1.347.661a6.743 6.743 0 01-1.286 1.794.75.75 0 11-1.06-1.06z" />
+</svg>`;
+export const userIcon = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6 user_icon">
+<path fill-rule="evenodd" d="M7.5 6a4.5 4.5 0 119 0 4.5 4.5 0 01-9 0zM3.751 20.105a8.25 8.25 0 0116.498 0 .75.75 0 01-.437.695A18.683 18.683 0 0112 22.5c-2.786 0-5.433-.608-7.812-1.7a.75.75 0 01-.437-.695z" clip-rule="evenodd" />
+</svg>
+`;
```
Comment (aleksanderwalczuk) [inline src/types.js:15]: In this file we define only the keys. For type variable's please use the notation of uppercase combined with underscores as the example provided in the task description.

**Review by aleksanderwalczuk — CHANGES_REQUESTED** [review]
Fix it quick and let's move on

**Comment by Afanasi0101** [comment]
What u think about it ?

File: src/components/btn.js [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
-  base: 'btn base_btn',
-  smallBase: 'btn base_btn_small',
-  baseLarge: 'btn base_btn_large',
-  nav: 'nav',
+  PRIMARY_BTN: 'btn primary_btn',
+  PRIMARY_BTN_LARGE: 'btn primary_btn_large',
+  BASE_BTN: 'btn base_btn',
+  SMALL_BASE_BTN: 'btn base_btn_small',
+  LARGE_BASE_BTN: 'btn base_btn_large',
+  NAV_BTN: 'nav',
```
Comment (aleksanderwalczuk) [inline src/components/btn.js:9]: Why only lg base btn is imported?

File: src/components/icon.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
   <path d="M13.06 15.473a48.45 48.45 0 017.666-3.282c.134 1.414.22 2.843.255 4.285a.75.75 0 01-.46.71 47.878 47.878 0 00-8.105 4.342.75.75 0 01-.832 0 47.877 47.877 0 00-8.104-4.342.75.75 0 01-.461-.71c.035-1.442.121-2.87.255-4.286A48.4 48.4 0 016 13.18v1.27a1.5 1.5 0 00-.14 2.508c-.09.38-.222.753-.397 1.11.452.213.901.434 1.346.661a6.729 6.729 0 00.551-1.608 1.5 1.5 0 00.14-2.67v-.645a48.549 48.549 0 013.44 1.668 2.25 2.25 0 002.12 0z" />
   <path d="M4.462 19.462c.42-.419.753-.89 1-1.394.453.213.902.434 1.347.661a6.743 6.743 0 01-1.286 1.794.75.75 0 11-1.06-1.06z" />
 </svg>
 `,
-  user: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6 user_icon">
+  USER_ICON: `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6 user_icon">
```
Comment (aleksanderwalczuk) [inline src/components/icon.js:8]: You should add imports

File: src/types.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+export const PRIMARY_BTN = 'primary_btn';
+export const PRIMARY_BTN_LARGE = 'primary_btn_large';
+export const BASE_BTN = 'base_btn';
+export const SMALL_BASE_BTN = 'base_btn_small';
+export const LARGE_BASE_BTN = 'base_btn_large';
+export const NAV_BTN = 'nav';
```
Comment (aleksanderwalczuk) [inline src/types.js:6]: I think nav isn't the best class name for a navigation button

File: src/types.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+<path d="M13.06 15.473a48.45 48.45 0 017.666-3.282c.134 1.414.22 2.843.255 4.285a.75.75 0 01-.46.71 47.878 47.878 0 00-8.105 4.342.75.75 0 01-.832 0 47.877 47.877 0 00-8.104-4.342.75.75 0 01-.461-.71c.035-1.442.121-2.87.255-4.286A48.4 48.4 0 016 13.18v1.27a1.5 1.5 0 00-.14 2.508c-.09.38-.222.753-.397 1.11.452.213.901.434 1.346.661a6.729 6.729 0 00.551-1.608 1.5 1.5 0 00.14-2.67v-.645a48.549 48.549 0 013.44 1.668 2.25 2.25 0 002.12 0z" />
+<path d="M4.462 19.462c.42-.419.753-.89 1-1.394.453.213.902.434 1.347.661a6.743 6.743 0 01-1.286 1.794.75.75 0 11-1.06-1.06z" />
+</svg>`;
+export const USER_ICON = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6 user_icon">
+<path fill-rule="evenodd" d="M7.5 6a4.5 4.5 0 119 0 4.5 4.5 0 01-9 0zM3.751 20.105a8.25 8.25 0 0116.498 0 .75.75 0 01-.437.695A18.683 18.683 0 0112 22.5c-2.786 0-5.433-.608-7.812-1.7a.75.75 0 01-.437-.695z" clip-rule="evenodd" />
+</svg>
+`;
```
Comment (aleksanderwalczuk) [inline src/types.js:15]: We should keep only key names in here.

**Review by aleksanderwalczuk — CHANGES_REQUESTED** [review]
Looks better but there are still few changes to be made

File: src/types.js (hunk 7 above)
Comment (Afanasi0101) [reply src/types.js:15]: What do you mean ? 
1. export const HAT = `svg...` ;
2. export const HAT_ICON

File: src/types.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+<path d="M13.06 15.473a48.45 48.45 0 017.666-3.282c.134 1.414.22 2.843.255 4.285a.75.75 0 01-.46.71 47.878 47.878 0 00-8.105 4.342.75.75 0 01-.832 0 47.877 47.877 0 00-8.104-4.342.75.75 0 01-.461-.71c.035-1.442.121-2.87.255-4.286A48.4 48.4 0 016 13.18v1.27a1.5 1.5 0 00-.14 2.508c-.09.38-.222.753-.397 1.11.452.213.901.434 1.346.661a6.729 6.729 0 00.551-1.608 1.5 1.5 0 00.14-2.67v-.645a48.549 48.549 0 013.44 1.668 2.25 2.25 0 002.12 0z" />
+<path d="M4.462 19.462c.42-.419.753-.89 1-1.394.453.213.902.434 1.347.661a6.743 6.743 0 01-1.286 1.794.75.75 0 11-1.06-1.06z" />
+</svg>`;
+export const USER = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" class="w-6 h-6 user_icon">
+<path fill-rule="evenodd" d="M7.5 6a4.5 4.5 0 119 0 4.5 4.5 0 01-9 0zM3.751 20.105a8.25 8.25 0 0116.498 0 .75.75 0 01-.437.695A18.683 18.683 0 0112 22.5c-2.786 0-5.433-.608-7.812-1.7a.75.75 0 01-.437-.695z" clip-rule="evenodd" />
+</svg>
+`;
```
Comment (aleksanderwalczuk) [inline src/types.js:15]: As values add `icon hat-icon` and following for the other one

**Review by aleksanderwalczuk — CHANGES_REQUESTED** [review]
Few more to go

File: src/components/contentWrapper.js [hunk 9]
```diff
@@ -1,15 +1,16 @@
...
+import { LARGE_BASE_BTN, USER_ICON } from '../types';
 
 function primaryAction() {
   console.log('btn clicked!');
 }
 
 function createContentWrapper() {
   const contentWrapper = document.createElement('div');
-  const primaryBtn = createBtn('play the game', 'primary', primaryAction);
+  const primaryBtn = createBtn('play the game', LARGE_BASE_BTN, primaryAction);
```
Comment (aleksanderwalczuk) [inline src/components/contentWrapper.js:11]: why `primaryBtn` has type of `LARGE_BASE_BTN`? 🤔

File: src/types.js (hunk 7 above)
Comment (aleksanderwalczuk) [reply src/types.js:15]: https://github.com/bright-bounty/quiz/pull/14#discussion_r990245010

File: src/components/btn.js [hunk 10]
```diff
@@ -1,10 +1,14 @@
+import {
+  LARGE_BASE_BTN, PRIMARY_BTN, PRIMARY_BTN_LARGE, BASE_BTN, SMALL_BASE_BTN, NAV,
```
Comment (aleksanderwalczuk) [inline src/components/btn.js:2]: NAV isn't the best name for a button type ;)

File: src/types.js (hunk 7 above)
Comment (aleksanderwalczuk) [reply src/types.js:15]: https://github.com/bright-bounty/quiz/pull/14#discussion_r990245010

**Review by aleksanderwalczuk — APPROVED** [review]
Lgtm

**Pull request merged**